src/App.js

```diff
@@ -28,9 +28,9 @@ class Quotes extends React.Component {
   render() {
     return (
       <div className="quoteBox">
-        <p className="time"></p>
-        <p>{quotes[0].content}</p>
-        <p>- {quotes[0].film}</p>
+        <p className="time">{this.time}</p>
+        <p>{quotes[this.random].content}</p>
+        <p>- {quotes[this.random].film}</p>
```

Comment:
1) you're accessing quotes twice 
You could have this to be stored in a variable for clarity and performance

2) I struggle to understand where `this.random` was coming from
This would be a more "react" way of doing 

```


class Quotes extends React.Component {
  _newQuote() {
    this.setState({
      quote: quotes[Math.floor(Math.random * quotes.length)]
    })
  }
  render() {
    const {quote} = this.state
    return (
      <div className="quoteBox">
        <p className="time">{this.time}</p>
        <p>{quote.content}</p>
        <p>- {quote.film}</p>
        <button onClick={() => this._newQuote()}>New Quote</button>
      </div>
    )
  }
}